exec.go

```diff
@@ -210,5 +212,20 @@ func execute(params execParams) error {
 		return err
 	}
 
+	if !params.detach {
+		p, err := os.FindProcess(process.Pid)
+		if err != nil {
+			return err
+		}
+
+		ps, err := p.Wait()
```

Comment:
Yes exactly, but it was not an issue from Docker perspective because Docker handle this on his own.